pkg/fakerp/fakerp.go

```diff
@@ -142,34 +142,19 @@ func createOrUpdate(ctx context.Context, log *logrus.Entry, cs, oldCs *api.OpenS
 		return nil, err
 	}
 
-	// persist the OpenShift container service
-	log.Info("persist config")
-	bytes, err := yaml.Marshal(cs)
-	if err != nil {
-		return nil, err
-	}
-	dataDir, err := shared.FindDirectory(shared.DataDirectory)
-	if err != nil {
-		return nil, err
-	}
-	err = ioutil.WriteFile(filepath.Join(dataDir, "containerservice.yaml"), bytes, 0600)
-	if err != nil {
-		return nil, err
-	}
-
-	err = os.MkdirAll(filepath.Join(dataDir, "_out"), 0777)
+	err = acceptMarketplaceAgreement(ctx, log, cs, testConfig)
 	if err != nil {
 		return nil, err
 	}
 
 	// write out development files
 	log.Info("write helpers")
-	err = writeHelpers(cs)
+	err = os.MkdirAll(filepath.Join(shared.PkgRoot+"/_data/_out"), 0777)
```

Comment:
why use filepath.Join at all if you're concatenating the path here?

Reply:
thanks - slip